src/Pate/Verification/Widening.hs

```diff
@@ -212,42 +214,63 @@ localWideningGas :: Gas
 localWideningGas = Gas 100
 
 
-widenPostcondition :: forall sym arch.
-  PA.ValidArch arch =>
+--TODO: this is a bit of a cludge - we should standardize how EquivM interacts
+-- with the online solver process
+withPredomain ::
```

Comment:
We should probably document what `withPredomain` means. What does this introduce into the assumption environment?

Reply:
Added a comment.